scripts/auto-config_UWARL_catkin_ws.zsh

```diff
@@ -16,9 +16,9 @@ else
     ic_err " ERROR::ROS does not exist, maybe you have not yet installed ROS?"
     ### Attempt to install ROS:
     case $LOCAL_DISTRIB_CODENAME in
-        "focal" )
-            ic_wrn "Attempting to auto-install ROS Noetic for Jetson!"
-            install_ros_noetic
+        "bionic" )
+            ic_wrn "Attempting to auto-install ROS melodic for Jetson!"
+            install_ros_melodic
```

Comment:
Please revert this, this does not install melodic, you have to define your own `install_ros_melodic`.
However, I implemented `install_ros_noetic` for noetic. @CazesC 
This will not install noetic as long as your os is not Ubuntu 20 focal.